DEPS

```diff
@@ -88,6 +88,7 @@ vars = {
   'dart_stack_trace_tag': '1.9.2',
   'dart_stream_channel_tag': '1.6.4',
   'dart_string_scanner_tag': '1.0.2',
+  'dart_term_glyph_tag': '1.0.0',
```

Comment:
Was this change intentional? It seems to be unrelated to the generation of .packages file.

Reply:
Yes. This is a package needed to run the test. (I think test package depends on it).

It used to be pulled by `pub get`.

Reply:
Ah, interesting.